include/ProgramRepresentation/DisjointedPVAliasSets.h

```diff
@@ -0,0 +1,86 @@
+#ifndef DISJOINED_PV_ALIAS_SETS_H
+#define DISJOINED_PV_ALIAS_SETS_H
+
+#include "ProgramRepresentation/PVAliasSet.h"
+
+// manages a set of alias sets
```

Comment:
How about this: "manages a disjoint set of program variable aliases. each program variable is an element of no more than 1 set." I am a little uncertain for a class comment how much detail I need to get into, or what specifically I need to mention

Reply:
I would include a link to the wiki article (if it is that data structure), so that anyone who isn't familiar with it knows what you mean. 

> I am a little uncertain for a class comment how much detail I need to get into, or what specifically I need to mention

The important thing here is precision: you need to precisely specify what it is the class does. You can do that at a high-level with few words (e.g., by using technical terms or jargon, or by referring to outside resources like the wiki page that describe the data structure in more detail). Your goal in a class comment should be to communicate to the reader what the purpose of the class is.